<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTileDialog</class>
 <widget class="QDialog" name="AddTileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>57</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="checo::ui::MultiIconButton" name="btnSelectFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>File</string>
     </property>
     <property name="icon">
      <iconset theme="file"/>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="checo::ui::MultiIconButton" name="btnSelectDirectory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Folder</string>
     </property>
     <property name="icon">
      <iconset theme="directory"/>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="checo::ui::MultiIconButton" name="btnSelectApplication">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Application</string>
     </property>
     <property name="icon">
      <iconset theme="application"/>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>checo::ui::MultiIconButton</class>
   <extends>QPushButton</extends>
   <header>checo/ui/multi_icon_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
